packages/app/src/cli/models/app/app.test-data.ts

```diff
@@ -309,6 +310,30 @@ export async function testFunctionExtension(
   return extension
 }
 
+interface TestPaymentsAppExtensionOptions {
+  dir?: string
+  config: PaymentsAppExtensionConfigType
+  entryPath?: string
+}
+export async function testPaymentsAppExtension(
+  opts: TestPaymentsAppExtensionOptions,
+): Promise<ExtensionInstance<PaymentsAppExtensionConfigType>> {
+  const directory = opts.dir ?? '/tmp/project/extensions/my-payments-app-extension'
```

Comment:
We copying what other extensions are doing. Since this is also for testing, I don't think we should create such indirection. The directory is clear, and its purpose makes sense as long as the target directory is visible within the method.

Reply:
Yeah, you can keep it like this 👌